how-to/how-to-design-database-schema.md

```diff
@@ -0,0 +1,304 @@
+---
+title: How to design your Telerik Platform mobile app backend database
+description: An overview of NoSQL database vs relational database and how to structure your data in Telerik Platform.
+type: how-to
+page_title: Telerik Platform Schema Design
+slug: design-database-schema
+position:
+tags: backendservices, relations
+ticketid:
+pitsid:s
+
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>
+		{{site.bs-d}} ({{site.bs}}), <!--Data (Backend Services)-->
+	</td>
+  </tr>
+</table>
+
+## Description
+- You are not sure how to organize your data?
+- Should you normalize the data (split the data into multiple content types) or not (consolidate data into fewer content types)?
+- What are the best practices in designing the backend database schema of your mobile app?
+- Is NoSQL different than a relational database?
+- Having trouble retrieving the data from multiple content types?
+
+If you are asking yourself one of these questions, then this article might be useful.
+
+## Solution
+The structure of the data inside your data storage is important because it determines how quickly and efficiently data is pulled out of it and how easily you can query the data and use advanced techniques like aggregation.
+
+Telerik Platform uses MongoDB as a database provider. MongoDB is a NoSQL database which is quite different from a relational database. This means that the best database schema for your Telerik Platform app will not necessarily follow the same principles as with a relational database.
+
+#### SQL Tables vs. NoSQL Documents
+
+SQL databases are designed to store related data in tables. Assume you have a Facebook or Twitter type of social app where people share personal moments with their friends. You may have a table named `Posts` where all user posts are stored:
+
+Table `Posts`
+
+Id | Text | User |CreatedAt
+--------|--------|--------|----
+0 | It is finally time for graduation! Good job everyone, we made it.  | Michael | Aug 03, 2016, 01:08 PM
+1 | The most amazing sunset I have ever seen at Phuket, Thailand | Seth | Aug 01, 2016, 01:08 PM
+
+
+Every row represents a different post record. The structure of a relational table is fixed - you cannot store fields different from the one that is expected or insert a string where a number is expected.
+
+A NoSQL database does not store data in tables - it stores them in collections. In contrast to a relational database table, a collection does not impose fixed rules what information can be stored in it. Telerik Platform calls these collections "content types" because they are designed to store similar content - in other words, the content of a single type. In addition, the name emphasizes on the fact that the content type is not an SQL table.

```

Comment:
"because they are designed to store similar content." should be enough. You can omit: 'in other words '